Sources/Kevin/Core/Styles/KevinTheme.swift

```diff
@@ -157,8 +129,6 @@ open class KevinTheme {
     open var listTableStyle = ListTableStyle()
     open var navigationLinkStyle = NavigationLinkStyle()
     open var mainButtonStyle = MainButtonStyle()
-    open var negativeButtonStyle = NegativeButtonStyle()
-    open var textFieldStyle = TextFieldStyle()
```

Comment:
Are we sure we want to remove these fields in this PR? It's a breaking change that might affect everyone using the SDK.

Reply:
Can't say I am 100% sure on this, but this is such a minor breaking change. Those fields weren't used anywhere in the code, so deleting them doesn't affect anything and fixing this from developer side is a matter of deleting some Theme lines if they overrode them in first place